frontend/src/scenes/sessions/SessionDetails.tsx

```diff
@@ -90,6 +93,23 @@ export function SessionDetails({ session }: { session: SessionType }): JSX.Eleme
                 },
                 rowExpandable: (event) => !!event,
                 expandRowByClick: true,
+                columnWidth: ANTD_EXPAND_BUTTON_WIDTH + MATCHING_EVENT_ICON_SIZE,
+                expandIcon: function _renderExpandIcon(expandProps) {
+                    const { record: event } = expandProps
```

Comment:
Probably worth doing this above? E.g. `function _renderExpandIcon({ record }) {

Reply:
Hm I did this because I still need the whole expandProps object in line 100

Reply:
Didn't catch that, my bad!

Reply:
no worries!